content/en/agent/proxy.md

```diff
@@ -25,6 +28,10 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 2. Using HAProxy (if you want to proxy **more than 16-20 Agents** through the same proxy)
 3. Using the Agent as a proxy (for **up to 16 Agents** per proxy, **only on Agent v5** )
 
+## FIPS Compliance
+
+For information on setting up a FIPS proxy with the Datadog Agent, see [Datadog FIPS Compliance][8]. The FIPS proxy is only available in the US1-FED region.
```

Comment:
It should be noted here that a regular proxy is not supported while also using the FIPS proxy